packages/cli/src/oclif/commands/scaffold.js

```diff
@@ -1,181 +1,116 @@
-        ].join('\n')
-      );
-    }
-
-    lines.splice(linesDefIndex + 1, 0, '    ' + injectorLine);
-    return lines.join('\n');
-  }
+    // * rewire the index.js to point ot the new file
+    this.startSpinner(`Rewriting your ${entry}`);
 
-  async writeTemplateFile(type, templateContext, dest) {
-    const templatePath = getTemplatePath(type);
-    const destPath = path.join(process.cwd(), `${dest}.js`);
-    const preventOverwrite = !this.flags.force;
+    const entryFilePath = path.join(process.cwd(), entry);
+    await updateEntryFile(
+      entryFilePath,
+      getVariableName(actionType, noun),
+      getFilename(newActionDir, noun),
+      templateContext.KEY,
+      actionType
+    );
 
-    if (preventOverwrite && fileExistsSync(destPath)) {
-      const [filename, location] = splitFileFromPath(destPath);
-
-      return this.error(
-        [
-          `File ${colors.bold(filename)} already exists within ${colors.bold(
-            location
-          )}.`,
-          'You could:',
-          '  1. Choose a different filename',
-          `  2. Delete ${filename} from ${location}`,
-          `  3. Run ${colors.italic('scaffold')} with ${colors.bold(
-            '--force'
-          )} to overwrite the current ${filename}`
-        ].join('\n')
-      );
-    }
-
-    const template = await readFile(templatePath);
-    const renderTemplate = _.template(template.toString(), {
-      interpolate: /<%=([\s\S]+?)%>/g
-    });
-
-    this.startSpinner(`Writing new ${dest}.js`);
-
-    await ensureDir(path.dirname(destPath));
-    await writeFile(destPath, renderTemplate(templateContext));
     this.stopSpinner();
-  }
 
-  async perform() {
-    const { name, type } = this.args;
-
-    if (!typeMap[type]) {
-      return this.error(
-        `Scaffold type "${type}" not found! Please see \`zaper help scaffold\`.`
-      );
-    }
-
-    try {
-      const { defaultDest, ...templateContext } = createTemplateContext(
-        type,
-        name
-      );
-      const { dest = defaultDest, entry = 'index.js' } = this.flags;
-      const entryFile = path.join(process.cwd(), entry);
-
-      this.log(`Adding ${type} scaffold to your project.\n`);
-
-      await this.writeTemplateFile(type, templateContext, dest);
-      await this.writeTemplateFile('test', templateContext, `test/${dest}`);
-
-      this.startSpinner(`Rewriting your ${entry}`);
-
-      const updatedEntry = await this.updateEntry(
-        entryFile,
-        entry,
-        templateContext.CAMEL,
-        dest
-      );
-
-      await writeFile(entryFile, updatedEntry);
-      this.stopSpinner();
-
-      this.log(
-        '\nFinished! We did the best we could, you might gut check your files though.'
-      );
-    } catch (error) {
-      this.error(error);
-    }
+    this.log(`\nFinished! Your new ${actionType} is ready to use.`);
   }
 }
 
 ScaffoldCommand.args = [
   {
-    name: 'type',
-    help: 'What type of thing are you creating',
+    name: 'actionType',
```

Comment:
this is non-breaking since this just describes the variable we access this at in the code